docs/src/pages/versions/versions.md

```diff
@@ -84,11 +84,11 @@ The open-source community is always welcome to submit new features and bug fixes
 The current status of each MUI version is as follows:
 
 - MUI Core v5: ✅ Active development and continuous support.
-- [MUI Core v4](https://v4.mui.com/): ⚠️ Guaranteed Support (only) for security issues and regressions.
-- [MUI Core v3](https://v3.mui.com/): 🅧 No longer supported.
-- ~MUI Core v2 (never existed)~.
-- [MUI Core v1](https://v1.mui.com/): 🅧 No longer supported.
-- [MUI Core v0.x](https://v0.mui.com/#/): 🅧 No longer supported.
+- [Material UI v4](https://v4.mui.com/): ⚠️ Guaranteed Support (only) for security issues and regressions.
```

Comment:
```suggestion
- [Material UI v4](https://v4.mui.com/): ⚠️ Guaranteed Support for security issues and regressions.
```
What sort of regressions do you anticipate?

Reply:
> What sort of regressions do you anticipate?

An example from Chrome. They https://github.com/mui/material-ui/issues/24519 broke v4.